src/Concordium/Client/Runner.hs

```diff
@@ -1200,7 +1200,7 @@ encryptedTransferTransactionConfirm EncryptedTransferTransactionConfig{..} confi
         = ettTransactionCfg
 
   logInfo
-    [ printf "transferring %s GTU from encrypted balance of account %s to %s" (Types.amountToString ettAmount) (showNamedAddress addr) (showNamedAddress ettReceiver)
+    [ printf "transferring %s CCD from encrypted balance of account %s to %s" (Types.amountToString ettAmount) (showNamedAddress addr) (showNamedAddress ettReceiver)
```

Comment:
```suggestion
    [ printf "transferring %s CCD from shielded balance of account %s to %s" (Types.amountToString ettAmount) (showNamedAddress addr) (showNamedAddress ettReceiver)
```

Also, we could use `showCcd` consistently here and several places below.

Reply:
One thing at a time. :)